src/docfx/build/Build.cs

```diff
@@ -40,6 +40,11 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Repo
                 Legacy.ConvertToLegacyModel(docset, context, files, sourceDependencies, tocMap);
             }
 
+            if (!docset.Config.Output.Json && docset.Config.Dependencies.ContainsKey("_themes"))
```

Comment:
>docset [](start = 17, length = 6)

options.Legacy &&

Reply:
Right, this could be merged with LegacyMetadata